core/src/modules/ops/runs.ts

```diff
@@ -276,6 +277,9 @@ export namespace RunOps {
       if (payload.class === taskName) found = found.concat(payload);
     }
 
-    return found.filter((t) => t.args[0].runId === runId).length > 0;
+    return (
+      found.filter((t: TaskInputs[number]) => t.args[0].runId === runId)
+        .length > 0
```

Comment:
Nitpick: Don't need the `length > 0` since a length of 0 is falsy.

Reply:
I think it's needed as we want this method to return a boolean. Otherwise, the type isn't defined